xlators/mgmt/glusterd/src/glusterd-messages.h

```diff
@@ -320,7 +320,8 @@ GLFS_MSGID(
     GD_MSG_LSTAT_FAIL, GD_MSG_VOLUME_NOT_IMPORTED,
     GD_MSG_ADD_BRICK_MNT_INFO_FAIL, GD_MSG_GET_MNT_ENTRY_INFO_FAIL,
     GD_MSG_QUORUM_CLUSTER_COUNT_GET_FAIL, GD_MSG_POST_COMMIT_OP_FAIL,
-    GD_MSG_POST_COMMIT_FROM_UUID_REJCT, GD_MSG_POST_COMMIT_REQ_SEND_FAIL);
+    GD_MSG_GRACEFUL_CLEANUP_SET_FAIL, GD_MSG_POST_COMMIT_FROM_UUID_REJCT,
```

Comment:
New messages must be added to the end of the list. Otherwise the id's of following messages will be changed from version to version.